code/src/components/Activity.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const activechoice = ['running 🏃🏻', 'breathing 🌬', 'stretching 🧘🏽', 'walking 🚶🏽', 'biking 🚵🏽'];
+
+export const Activity = ({ activity, setActivity, step }) => {
+  return (
+    <>
+      <div className="content-container">
```

Comment:
I notice some of the classnames aren't in camel case. Not a biggie, but maybe for future reference?